examples/lib/stories/experimental/layout_component_example_size.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flame/components.dart';
+
+enum LayoutComponentExampleSize {
+  shrinkWrap(null, null),
+  small(640, 480),
+  large(1080, 720);
+
+  const LayoutComponentExampleSize(
+    this.x,
+    this.y,
+  );
+  final double? x;
+  final double? y;
```

Comment:
can this use shining new NullableVector2 ?

Reply:
One might actually say this should be a Vector2. But neither Vector2 nor NullableVector2 can be used here, because neither of them have const constructors. They need to be const, because used with the `LayoutComponentExampleSize` enum.